developers/bindings/thing-xml.md

```diff
@@ -411,7 +408,7 @@ public class ExampleHandlerFactory extends BaseThingHandlerFactory {
 
 ### Channel Categories
 
-A description about channel categories as well as an overview about which categories exist can be found in out [categories overview](../../concepts/categories.html).
+Channel categories are used to set an icon for that Channel. The available categories correspond with the available icons of the classic icon set, however categories are written in Java class-naming style, e.g. FrontDoor instead of lowercase frontdoor.
```

Comment:
```suggestion
Channel categories are used to provide meta information about channels. Channel categories describe the **functional purpose** of the channel and are used by the UI to render icons.
The available categories correspond with the [available icons of the classic iconset]({{base}}/configuration/iconsets/classic/), however categories are written in Java class-naming style, e.g. `BatteryLevel` instead of lowercase `batterylevel`.
```